packages/playground/common/src/index.ts

```diff
@@ -45,14 +47,23 @@ export const unzipFile = async (
             $zip = new ZipArchive;
             $res = $zip->open($zipPath);
             if ($res === TRUE) {
-                $zip->extractTo($extractTo);
-                $zip->close();
-                chmod($extractTo, 0777);
+				for ($i = 0; $i < $zip->numFiles; $i++) {
+					$filename = $zip->getNameIndex($i);
+					$fileinfo = pathinfo($filename);
+					$extractFilePath = $extractTo . DIRECTORY_SEPARATOR . $filename;
```

Comment:
Let's trim any trailing `$extractTo` doesn't end with `DIRECTORY_SEPARATOR`.

Reply:
Also, I'm not sure if `DIRECTORY_SEPARATOR` is any useful these days. Windows seems to understand forward slashes just fine, and in WASM we're assuming linux anyway.